.github/workflows/conformance-e2e.yaml

```diff
@@ -348,7 +348,7 @@ jobs:
 
             mkdir -p cilium-junits
 
-      - name: Run tests (${{ join(matrix.*, ', ') }})
+      - name: Run tests
```

Comment:
As far as I am concerned step name should match the --junit-property github_job_step name so either step name should be `Run tests (${{ matrix.name }})` or --junit-property should be `github_job_step="Run tests"`

Reply:
TIL, thanks! I changed --junit-property to `Run tests`